packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -253,6 +253,19 @@ export function buildWebpackBrowser(
       }
 
       const useBundleDownleveling = isDifferentialLoadingNeeded && !options.watch;
+
+      // disable appending sourcemap comments if using bundle downleveling
+      // not doing so will result in double sourcemap comments and a large increase in memory usage
+      // NOTE: Once the webpack config is refactored this should be done directly on the options
+      if (useBundleDownleveling && config.plugins) {
+        const plugin = config.plugins.find(p => p instanceof webpack.SourceMapDevToolPlugin);
```

Comment:
Why not set the sourcemap option to hidden, which will have the same result.

https://github.com/angular/angular-cli/blob/862b28f844cddcc406defac6a32d6e266221d499/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/utils.ts#L102

Reply:
Updated.  I was hoping to avoid adding more differential loading special casing to the webpack configs themselves.  But they are going to be refactored so it should be ok for now.